src/navigation/_ComponentToolbar.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import Button from 'terra-button';
+import IconLeftPane from 'terra-icon/lib/icon/IconLeftPane';
+import MenuButton from '../menu-button/_MenuButton';
+
+import styles from './ComponentToolbar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Callback for showing the side menu
+   */
+  onToggle: PropTypes.func,
+
+  /**
+   * Menu visible external state
+   */
+  menuIsVisible: PropTypes.bool,
+
+  /**
+   * Config for the theme and locale menu button.
+   */
+  devToolsConfig: PropTypes.shape({
+    selectedTheme: PropTypes.string,
+    selectedLocale: PropTypes.string,
+    themes: PropTypes.arrayOf(PropTypes.string),
+    locales: PropTypes.arrayOf(PropTypes.string),
+    onChangeLocale: PropTypes.func,
+    onChangeTheme: PropTypes.func,
+  }),
+};
+
+const defaultProps = {
+  onToggle: undefined,
+  menuIsVisible: true,
```

Comment:
Can we update for the default value to be false to align with our boolean default prop convention or rename to for the negative, i.e. `hideMenu: false`